src/app/content/components/NudgeStudyTools/index.tsx

```diff
@@ -33,19 +33,38 @@ import {
   usePositions,
 } from './utils';
 
+function NudgeStudyTools() {
+  const isMobile = useMatchMobileMediumQuery();
+  const positions = usePositions(isMobile);
+
+  if (!positions) { return null; }
+
+  return (
+    <NudgeStudyTools2 positions={positions} />
+  );
+}
+
 // tslint:disable-next-line: variable-name
-const NudgeStudyTools = () => {
+const NudgeStudyTools2 = ({
```

Comment:
This seems harder to read, is splitting out this component necessary since we have the early return? Alternatively, maybe we adjust the var name, this makes me think it's a versioned component at first glance.

Reply:
The reason for splitting is to allow the short-circuit to avoid all the hooks that are associated with the real component. Sometimes I name these pretesting components like MaybeNudgeStudyTools. In this case, I'm just moving the test up to the caller that already had another short-circuit defined.

Reply:
You can't return before hooks or you get warnings about [conditional hooks](https://dev.to/lexlohr/conditional-hooks-1fpj).

Reply:
Haha I never remember this until the linter yells at me about it, thanks for the reminder :)